stroyprombeton/tests/tests_views.py

```diff
@@ -773,3 +773,32 @@ def test_non_existing_tags_404(self):
         })
         response = self.client.get(bad_tag_url)
         self.assertEqual(response.status_code, 404)
+
+    # @todo #362:60m Collapse too tags on category page.
+    #  See test below and parent task for details.
+    @unittest.expectedFailure
+    def test_too_many_tags_collapse(self):
+        """
+        Page should not contain too many tags.
+
+        If page contains too many tags, it should collapse them and show short label instead.
+        """
+        group = models.TagGroup.objects.first()
+        models.Tag.objects.filter(group=group).delete()
+        product = models.Product.objects.get(name='Product #10 of Category #0 of #3')
+        from_index, to_index = 1, 22
```

Comment:
Let's introduce some variable to determine if there are too many tags and, correct the test with respect to this